app/src/features/apiClient/screens/apiClient/components/clientView/components/Collection/components/CollectionsVariablesView/CollectionsVariablesView.tsx

```diff
@@ -33,6 +35,13 @@ export const CollectionsVariablesView: React.FC<CollectionsVariablesViewProps> =
     return setCollectionVariables(pendingVariables, collection.id)
       .then(() => {
         toast.success("Variables updated successfully");
+        trackVariablesUpdated({
+          type: "collection_variable",
+          updated_variable_count: getUpdatedVariableCount(
+            collectionVariables[collection.id]?.variables,
+            pendingVariables
+          ),
```

Comment:
We always dump the entire pendingVariables. No diff is required. 
`Object.entries(pendingVariables).length` should be enough.

We can have this event changed to trackVariablesSaved(type, num_variables=`<updated count of pending Variables>`)

cc: @echo-sg